src/app/ubs-admin/components/ubs-admin-table/table-cell-readonly/table-cell-readonly.component.scss

```diff
@@ -0,0 +1,16 @@
+div {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: 0 20px;
+}
+
+.readonly_date {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 20px;
+}
+
+.tool_tip {
+  font-size: 20px !important;
```

Comment:
Can we avoid !important here?
Maybe we can use :host or :ng-deep instead?
https://blog.angular-university.io/angular-host-context/

If not, let it be :)

Reply:
If not, let it be :)